src/main/java/frc/robot/commands/vision/VisionAim.java

```diff
@@ -57,27 +49,8 @@ public void initialize()
   @Override
   public void execute()
   {
-
-    double turn = robot.vision.aimAtTarget(); //Get the turn speed from the camera
-    if (turn == previousTurn) {
-      angleError +=  robot.drivetrain.gyroAngle() - currentAngle; 
-      currentAngle = robot.drivetrain.gyroAngle();
-    } else {
-      previousTurn = turn;
-      angleError = turn*PIXELS_TO_DEGREES;
-      currentAngle = robot.drivetrain.gyroAngle();
-    }
-
-    SmartDashboard.putNumber("Angle Error", angleError);
-    double turnSpeed = angleError*Drivetrain.AIM_TURN_SPEED;
-
-    if (Math.abs(turnSpeed) < MIN_TURN_SPEED) turnSpeed = Math.copySign(MIN_TURN_SPEED, turnSpeed);
-    if (Math.abs(turnSpeed) > MAX_TURN_SPEED) turnSpeed = Math.copySign(MIN_TURN_SPEED, turnSpeed);
-    if (Math.abs(angleError) > THRESHOLD) {
-      SmartDashboard.putBoolean("ALIGNED", false);
-      robot.drivetrain.moveMotors(0, turnSpeed);
-    } else {
-      SmartDashboard.putBoolean("ALIGNED", true);
+    if (!robot.vision.aligned) {
+      robot.drivetrain.moveMotors(0, robot.vision.turnSpeed);
```

Comment:
You can try running the calculation here and temporarily pass the gyro to vision.